packages/contracts/test/zrx_token.ts

```diff
@@ -73,8 +82,7 @@ contract('ZRXToken', (accounts: string[]) => {
             const receiver = spender;
             const initOwnerBalance = await zeroEx.token.getBalanceAsync(zrxAddress, owner);
             const amountToTransfer = new BigNumber(1);
-            const txHash = await zeroEx.token.transferAsync(zrxAddress, owner, receiver, amountToTransfer);
-            await zeroEx.awaitTransactionMinedAsync(txHash);
```

Comment:
Why is it no longer necessary to await the tx to be mined?

Reply:
Tests run only against testrpc. On testrpc we have instant mining and that's why we await transactions only if we need a receipt.